README.md

```diff
@@ -1,23 +1,45 @@
 ISO 15118 library suite
 =======================
 
-Currently, this is a proof of concept library.
+This is a C++ library implementation of ISO 15118 for part 20.
 
-Integration of ISO 15118 development for EVerest is under heavy development
-currently. Many features are already implemented, many are still to follow.
+Integration of ISO15118-20 development for EVerest is under heavy development
+currently. 
+
+## ISO 15118-20 Support
+
+The following tables show the current support for the listed EVSE ISO15118-20 features.
+
+| Feature                            | Supported          |
+|------------------------------------|--------------------|
+| TCP, TLS 1.2 & 1.3                 | :heavy_check_mark: |
+| DC, DC_BPT                         | :heavy_check_mark: |
+| AC, AC_BPT                         | WIP                |
+| WPT                                |                    |
+| ACDP                               |                    |
```

Comment:
Maybe we can provide the full name of some abbreviations as well. What do you think?

Reply:
:+1: I will add a list of abbreviations under this table